CIP-0030/README.md

```diff
@@ -66,6 +66,16 @@ type TransactionUnspentOutput = cbor<transaction_unspent_output>
 
 This allows us to use the output for constructing new transactions using it as an output as the `transaction_output` in the Shelley Multi-asset CDDL does not contain enough information on its own to spend it.
 
+
+### EnableOptions
+
+```
+type EnableOptions = {|
+  chainId?: cbor\<chain_id>
```

Comment:
Why use cbor here? I think we should be using dev/human-friendly types when possible (as with `address` that supports bech32 string in cip30).

I suggest to either change this to the "human-readable string" format that is already described in cip34, or make it support both formats.